tensorflow/tools/ci_build/osx/arm64/metal_plugin_test.py

```diff
@@ -0,0 +1,5536 @@
+#!/usr/bin/env python3
```

Comment:
@kulinseth Our internal presubmits require the Apache 2.0 license header ([example](https://github.com/tensorflow/tensorflow/blob/cded3794c001f11539c8749104648b942499e031/tensorflow/tools/ci_build/osx/arm64/tensorflow_as_build_nightly.Jenkinsfile#L1-L15)) to be present in almost all the files in OSS. Do you see any concerns to adding the header in here?